src/node/Main.hs

```diff
@@ -227,6 +237,8 @@ updateTriggerWorker = first pure $ worker mempty $ \_ -> do
 -- Wallet stuff
 ----------------------------------------------------------------------------
 
+#ifdef WITH_WALLET
+
```

Comment:
Why do you think e.g. `wallet` is not needed without `WITH_WALLET`? What about this https://github.com/input-output-hk/cardano-sl/pull/895/files#diff-816b9ae1342c0d73323326bc3391be25R129
Or `WITH_WEB` implies `WITH_WALLET`?